src/app/state/staking/saga.ts

```diff
@@ -200,6 +200,6 @@ export function* fetchAccount({ payload: address }: PayloadAction<string>) {
 }
 
 export function* stakingSaga() {
-  yield* takeEvery(stakingActions.fetchAccount, fetchAccount)
+  yield* takeLatest(stakingActions.fetchAccount, fetchAccount)
   yield* takeEvery(stakingActions.validatorSelected, getValidatorDetails)
```

Comment:
Race conditions elsewhere
- `stakingActions.validatorSelected` should be `takeLatest` - shows wrong info on race condition
- `networkActions.selectNetwork` would be more correct with `takeLatest` - switches to previous network on race condition 
- `ledgerActions.enumerateAccounts` with `takeLatest` should prevent Ledger crashes, but it doesn't https://github.com/oasisprotocol/oasis-wallet-web/issues/869